src/components/sharehouses-management/RotationCycles.tsx

```diff
@@ -80,7 +80,7 @@ export const RotationCycles = ({
 
   return (
     <>
-      {isSubmitting ? <LoadingSpinner isLoading={true} /> : ''}
+      {<LoadingSpinner isLoading={isSubmitting} />}
```

Comment:
Could you please delete the curly brackets?

Reply:
@HrRn 
Sorry! I deleted curly brackets.
`src/components/accordion-assignment-sheet/AccordionAssignmentSheet.tsx`
`src/components/sharehouses-management/RotationCycles.tsx`

Also, I found this file had the same isuue. Could you review this one too?
src/components/login/Form.tsx